home/utils.py

```diff
@@ -0,0 +1,84 @@
+import json
+import requests
+
+# from taggit.models import Tag
+from home.models import ContentPage, HomePage
+from wagtail.core import blocks
+from wagtail.images.models import Image
+from io import BytesIO
+from django.core.files.images import ImageFile
+from wagtail.images.blocks import ImageChooserBlock
+
+
+def import_content(file):
+    def get_body(message):
+        if message["attachment_media_type"] == "image":
+            im = ""
+            title = message["attachment_media_object"]["filename"]
+            try:
+                im = Image.objects.get(title=title).id
+            except Exception:
+                uri = requests.get(message["attachment_uri"])
+                image_file = ImageFile(BytesIO(uri.content), name=title)
+                image = Image(
+                    title=title,
+                    file=image_file,
+                    width=image_file.width,
+                    height=image_file.height,
+                )
+                image.save()
+                im = image.id
+
+            block = blocks.StructBlock(
+                [("message", blocks.TextBlock()), ("image", ImageChooserBlock())]
+            )
+            block_value = block.to_python({"message": message["answer"], "image": im})
+            return block_value
+        else:
+            block = blocks.StructBlock(
+                [
+                    ("message", blocks.TextBlock()),
+                ]
+            )
+            block_value = block.to_python({"message": message["answer"]})
+            return block_value
+
+    # def create_tags(tags, page):
+    #     for tag in tags:
+    #         created_tag, _ = Tag.objects.get_or_create(name=tag)
+    #         page.tags.add(created_tag)
+
+    ContentPage.objects.filter(id__lte=1000).delete()
```

Comment:
Why are we doing this in 3 lines?

Reply:
wagtail delete only allows 1000 at a time